Task-47220: Inconsistence of WYSIWYG in Table display

lbengayed opened the pull request:
When a table does have a long text, some words are cut and the line break is in the middle of the words.
This PR fix it

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

NaderSai committed:
Task-45373: Fix Notes application name and description are not displayed by their value in space settings interface (#144)

lbengayed committed:
Task-47220: Inconsistence of WYSIWYG in Table display

NaderSai committed:
Task-45373: Delete duplicated values (#149)

ahamdi commented on notes-service/src/main/resources/locale/portlet/Portlets_en.properties:
```diff
@@ -0,0 +1,2 @@
+SpaceSettings.application.notes.application.title=Notes Application
```
this file should be removed

ahamdi commented on notes-webapp/src/main/webapp/skin/less/notes/notes.less:
```diff
@@ -153,11 +153,19 @@
     min-height: calc(~"100vh - 170px");
   }
+  .notes-application-content{
+	overflow: auto;
```
Fix indentation of this file

ahamdi commented on notes-webapp/src/main/webapp/skin/less/notes/notes.less:
```diff
@@ -153,11 +153,19 @@
-	word-break: break-all;
-	border: 1px solid var(--allPagesBorderColor,#e1e8ee);
+	  border: 1px solid var(--allPagesBorderColor,#e1e8ee);
+	  border-right: none;
```
Consider RTLisation please

ahamdi commented on notes-webapp/src/main/webapp/skin/less/notes/notes.less:
```diff
@@ -153,11 +153,19 @@
+	  border-top: none;
+	  width: auto;
+	  td {
+		  border-right: 1px solid var(--allPagesBorderColor,#e1e8ee);
```
Consider RTLisation please

ahamdi reviewed (changes requested):
Please add a description for this PR

Rdidmohamedelhadi committed:
Task-47516 : For simple user, I can't add note (#158)

AzmiTouil committed:
Task-47653: Wrong error message when editing the name of note page (#160)

Wrong error message when editing the name of note page with a name that already exists

rdenarie reviewed (changes requested):
Hello,
Can you add a description following [our commit message best practises](https://developer.exoplatform.org/#id-commits-messages-best-practices)
It help us to understand what you want to do

lbengayed committed:
Task-47220: fix retour Inconsistence of WYSIWYG in Table display

lbengayed committed:
Update notes.less

rdenarie committed:
Merge branch 'feature/maintenance' into Task-47220
